cli/js/globals.ts

```diff
@@ -102,6 +102,8 @@ declare global {
 
     formatError: (e: Error) => string;
 
+    getPromiseDetails(promise: Promise<unknown>) : [number, unknown];
```

Comment:
This should be typed a lot more specifically.  If the second argument of the tuple is the value of the promise, then:

```suggestion
    getPromiseDetails<T>(promise: Promise<T>) : [number, T];
```

Also provide some JSDoc please, so this is easily documented.